keysign/discover.py

```diff
@@ -42,13 +42,17 @@ def start(self):
         if self.bt_code and not self.stopped:
             # We try Bluetooth, if we have it
             log.info("Trying to connect to %s with Bluetooth", self.bt_code)
-            self.bt = BluetoothReceive(self.bt_port)
+            # We try to see if Bluetooth was imported,
+            # else we log an event of missing Pybluez.
+            try:
+                self.bt = BluetoothReceive(self.bt_port)
+            except:
+                log.exception("PyBluez is probably missing.")
```

Comment:
If you do a thing like this, when an error occurs the control flow will execute the log instruction and then the `self.bt.find_key()`
But if there was an error, `self.bt` will be None and so another exception will be raised.

You are able to reproduce this bug (you are the one that reported it), so it is expected that you try the proposed fix in order to verify that, at least, it works in your machine and with your test case.

Reply:
I apologize. The error was in a different system. I forgot to make a check before updating.